srcpkgs/glibc/INSTALL

```diff
@@ -0,0 +1,8 @@
+# Build iconv module cache
+#
+case "${ACTION}" in
+post)
+	echo "Generating iconv module cache..."
+	iconvconfig --nostdlib --prefix="$(pwd)" usr/lib${XBPS_TARGET_WORDSIZE}/gconv
```

Comment:
That variable won't be defined. `usr/lib/gconv` is the best you can get, I think.